src/util/postgres_client.js

```diff
@@ -268,6 +300,14 @@ class PostgresTable {
         return {};
     }
 
+    async insertOneTransactional(transaction, data) {
```

Comment:
I would maybe add `_` to mark it as internal, we don't want to expose this as far as I understand.